eclipse-collections-api/src/main/java/org/eclipse/collections/api/RichIterable.java

```diff
@@ -2058,6 +2058,26 @@ default void appendString(Appendable appendable, String separator)
         return this.collectWith(function, parameter, target);
     }
 
+    /**
+     * This method will count the number of occurrences of each value calculated by applying the
+     * function to each element of the collection.
+     *
+     */
+    default <V> Bag<V> countByEach(Function<? super T, ? extends Iterable<V>> function)
```

Comment:
needs an @ since 10.0.0 tag
Please add the since tag for each interface level override.